dlt/common/destination/capabilities.py

```diff
@@ -76,6 +76,7 @@ class DestinationCapabilitiesContext(ContainerInjectableContext):
     # use naming convention in the schema
     naming_convention: TNamingConventionReferenceArg = None
     alter_add_multi_column: bool = True
+    create_table_not_exists: bool = True
```

Comment:
I suggest to rename it that it's clear it for `_dlt` tables:
For ex, `create_dlt_table_if_not_exists`

Reply:
this is in destination caps. but I will rename it to make clear what it means